### PR TITLE
feat(testing): add lint target for playwright

### PR DESCRIPTION
**Pull request by xiongemi**
## Related Issue(s)

Fixes #

### CODE REVIEW

**Review by mandarini — APPROVED** [review]
`e2e` changes look ok

File: packages/playwright/src/generators/configuration/files/playwright.config.ts.template [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { defineConfig } from '@playwright/test';
 import { nxE2EPreset } from '@nx/playwright/preset';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```
Comment (barbados-clemens) [inline packages/playwright/src/generators/configuration/files/playwright.config.ts.template:3]: could we conditionally add this eslint ignore comment when the webServerCommand and webServerAddress aren't included?

```suggestion
<% if(!webServerCommand || !webServerAddress) { %>// eslint-disable-next-line @typescript-eslint/no-unused-vars <% } %>
```

**Review by barbados-clemens — COMMENTED** [review]
small nitpick

**Review by barbados-clemens — APPROVED** [review]

File: packages/playwright/src/generators/configuration/files/playwright.config.ts.template (hunk 1 above)
Comment (xiongemi) [reply packages/playwright/src/generators/configuration/files/playwright.config.ts.template]: changed

**Pull request merged**